lib/process-services-cloud/src/lib/task/task-header/components/task-header-cloud.component.ts

```diff
@@ -230,6 +235,18 @@ export class TaskHeaderCloudComponent implements OnInit, OnDestroy, OnChanges {
                     key: 'id'
                 }
             ),
+            new CardViewTextItemModel(
+                {
+                    label: 'ADF_CLOUD_TASK_HEADER.PROPERTIES.PROCESS_INSTANCE_ID',
+                    value: this.processInstanceId,
+                    default: this.translationService.instant('ADF_CLOUD_TASK_HEADER.PROPERTIES.PROCESS_INSTANCE_ID_DEFAULT'),
+                    key: 'processInstanceId',
+                    clickable: true,
+                    clickCallBack: () => {
+                        this.redirectToProcessDetails();
```

Comment:
Adf component should not handle the navigation at all the application that uses this header component is responsible to navigate on click of the process instance Id.

Reply:
logic moved to the application level